programs/summerofcode/2024.md

```diff
@@ -31,6 +31,24 @@ You can find the project ideas from previous year [here](./2023.md).
 
 ### Proposals
 
+#### CNCF GOSST
+
+##### CNCF and Google Open Source Security Team GSoC Collaboration - Enhancing Security Across CNCF Ecosystem
+
+- Description: This project is a collaborative effort between the CNCF and Google's Open Source Security Team to improve security practices across various CNCF projects. The focus is identifying and addressing security vulnerabilities, integrating security tools like OSS-Fuzz, and enhancing build and release security processes. The goal is to get all CNCF projects to use scorecards (focusing on graduated/incubating projects first) and to remediate some of the findings.
```

Comment:
Who is providing the security expertise to support projects in remediating these findings? I really love this idea but I am deeply concerned that this will create additional work for projects that are not structured or experienced enough to intake these findings, triage, or remediate. Particularly as the scorecards indicators (and I admit it has been at least a year since I have evaluated them last) are not strong, still requires human review, and is yet another out of band activity for maintainers.  

How can we make this frictionless, a value-add, and add on to the projects' work?

Reply:
> Who is providing the security expertise to support projects in remediating these findings?

Dustin and other GOSST folks would act as the security experts -- We may shift the actual mentors as this work gets firmed up. I'll act as a liason between the mentee, GOSST folks and our projects.

> How can we make this frictionless, a value-add, and add on to the projects' work?

Part of the GSoC process is getting candidates to write proposals, and I think this concern is going to be a part of our selection criteria.